src/Microsoft.Health.Fhir.Core/Models/CodeableConceptInfo.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Core.Models
+{
+    public class CodeableConceptInfo
```

Comment:
Maybe named this something like `SystemValuePair`? `SystemAndValue`?

Reply:
Actually, shouldn't this be system and code so more like SystemCodePair? Then you can collapse CodingInfo into here as well.

Reply:
Merged both into `CodingInfo`